interfaces/IAnalytics.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "Module.h"
+
+// @stubgen:include <com/IIteratorType.h>
+
+namespace WPEFramework {
+namespace Exchange {
+    // @json 1.0.0 @text:keep 
+    struct EXTERNAL IAnalytics : virtual public Core::IUnknown {
+        enum { ID = ID_ANALYTICS };
+
+        virtual ~IAnalytics() = default;
```

Comment:
@adrianM27 one small remark ;-) As we are inheriting from Core::IUnknown, the destructor is already virtual. Using the override keyword the compiler will check that the base (destructor of Core::IUnknown) is virtual. Hence why I assumed you would drop the virtual keywrd and have the compiler check that the destructor of the base class is really virtual.
So suggest to state here: ~IAnalytics() override = default;

Reply:
Thanks @pwielders ,  updated